deeplabcut/generate_training_dataset/trainingsetmanipulation.py

```diff
@@ -832,16 +840,28 @@ def create_training_dataset(
 
     # Loading metadata from config file:
     cfg = auxiliaryfunctions.read_config(config)
+    dlc_root_path = auxiliaryfunctions.get_deeplabcut_path()
+    
+    if superanimal_name!="":
```

Comment:
I think all these suggestions can be applied at once by running black on the code.

```suggestion
    if superanimal_name != "":
```